crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -646,15 +646,15 @@ type Event {
 	data: MoveData!
 	"""
 	Representation of a Move value in JSON, where:
-	
+
```

Comment:
nit: I think you probably want to undo these changes?  Unfortunately, the schema generator leaves trailing whitespace, and your editor is probably cleaning it up when you open this file, but these will just get added back or cause a merge conflict if you check in the change.